MonoXAnalysis/cfg/run_monojet_cfg.py

```diff
@@ -22,14 +22,15 @@
 doT1METCorr = getHeppyOption("doT1METCorr",True)
```

Comment:
yes, do not change anything in the run_monojet_cfg.py. I thought to have reverted the code to the initial version